### PR TITLE
added cart context

### PR DESCRIPTION
**Pull request by tusharkandpal**
To check:
✅ added cart-context
✅ created CartCard & OrderCard components
✅ add, remove, quantity services for cart

Note: removed localStorage implementation for wishlist & cart since was creating conflicts.

### CODE REVIEW

File: src/components/card/cart-card/CartCard.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* || CART CARD */
+.private-component .card-horizontal {
+  margin: 0 0.8rem 0.8rem 0;
```
Comment (ShreyashMahajan) [inline src/components/card/cart-card/CartCard.css:3]: try to use px for less than 1rem

File: src/components/card/cart-card/CartCard.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import { useCart, useWishlist } from "../../../context/context";
+
+export const CartCard = (product) => {
+  const { _id, title, subTitle, imgUrl, originalPrice, sellingPrice, qty } =
+    product;
+  const { removeFromCartHandler, cartQuantityHandler } = useCart();
+  const { wishlist, addToWishlistHandler, removeFromWishlistHandler } =
+    useWishlist();
+
+  const discount = Math.round(
```
Comment (ShreyashMahajan) [inline src/components/card/cart-card/CartCard.jsx:11]: good to see that you are rounding off the discount

**Review by ShreyashMahajan — APPROVED** [review]
good to merge

**Review by ultronjs — APPROVED** [review]
Everything Looks well Structure.
Checked the UI also, looking clean
Cart Context is well written
Adding and removing functionality is also working fine
Good to merge

**Pull request merged**